src/index.js

```diff
@@ -52,7 +52,11 @@ const create = (pnConfig, ceConfig = {}) => {
 };
 
 // export the ChatEngine api
-module.exports = {
-    plugin: {}, // leave a spot for plugins to exist
-    create
+const ChatEngineCore = {
+    plugin: {},
+    create: create
 };
+
+module.exports = ChatEngineCore;
+
+module.exports.ChatEngineCore = ChatEngineCore;
```

Comment:
This doesn't seem right.

Reply:
this change is necessary in order to continue supporting the same point of entry and typescript, when was release the our sdk for angular2 we had some troubles of incompatibility and with this strategy we could resolve it

Reply:
yes I tried to use dts-gen when I was working on our sdk for angular2 but the result was not very good

Reply:
Your previous comment seems like it's in the wrong thread.

IRT "this change is necessary." Oh yeah, I have encountered stuff like this before.

Reply:
ahh yes I could reply in the correct thread lol

Reply:
This LGTM.